configs/ote/face-detection/face-detection-0200/template_experimental.yaml

```diff
@@ -0,0 +1,53 @@
+# Description.
+model_template_id: face-detection-0200
+name: Face Detection 0200
+task_type: DETECTION
+task_family: VISION
+instantiation: "CLASS"
+summary: Face Detection based on MobileNetV2 (SSD).
+application:
+  ~
```

Comment:
I guess it should be something like "Face detection". But we may leave it as is for now and get back to it later.

Reply:
Added